src/server/json_family_test.cc

```diff
@@ -656,7 +658,7 @@ TEST_F(JsonFamilyTest, ToggleLegacy) {
   ASSERT_THAT(resp, "OK");
 
   resp = Run({"JSON.TOGGLE", "json"});
-  EXPECT_THAT(resp, ErrArg("wrong JSON type of path value"));
+  EXPECT_THAT(resp, ErrArg("wrong number of arguments"));
```

Comment:
Interesting, the specification says that the path is an optional argument. But Redis also throws an error for the empty path
![image](https://github.com/user-attachments/assets/fe64e8b7-8eac-4070-ad33-b264ff239afe)